src/screens/guides/components/data-accessors/ecology.md

```diff
@@ -96,6 +96,44 @@ Data accessor props may be given as functions and used to process data, as in th
 ```
 
 
+## Sorting data
+
+
+Sorting can be applied to the final data via the dataSort prop. This prop must
+be a function acting as a standard JavaScript comparator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
+
+
+```playground
+  <VictoryChart
+    domainPadding={{x: 40}}
+  >
+    <VictoryBar
+      data={[
+        { experiment: 2, expected: 3.75, actual: 3.38 },
+        { experiment: 4, expected: 3.75, actual: 3.71 },
+        { experiment: 3, expected: 3.75, actual: 2.05 },
+        { experiment: 1, expected: 3.75, actual: 3.21 }
+      ]}
+      x="experiment"
+      y={(d) => (d.actual / d.expected) * 100}
+      dataSort={(a, b) => (a.experiment - b.experiment)}
```

Comment:
Here's a cool example that demonstrates why you would want to use this feature https://github.com/FormidableLabs/victory-chart/blob/master/demo/components/victory-line-demo.js#L304